### PR TITLE
Small fixes to README file

### PR DESCRIPTION
**Pull request by scottcwilson**
- Adds a step to Contributor instructions for those modifying commands.
- Changes forum link to http (https does not work)
- Adds "Sign the forum release thread" to contributor instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ Use the [installation instructions in the Hugo documentation](http://gohugo.io/o
 
     Go expects all of your libraries to be found in`$GOPATH`.
 
+7. If you are changing files in the `commands` directory, you will want to make a local change to `main.go` so that it imports from `./commands` instead of github.
```
Comment (bep) [inline README.md]: This isn't need - and may lead to faulty commits. 

The trick I use is to just clone spf13's Hugo (or go get) into `$GOPATH/src/github.com`, then add my Hugo fork as a remote. The imports are unchanged and everything just works as expected.

**Comment by spf13** [comment]
The other two changes are welcome though. Can you update the PR?

**Comment by scottcwilson** [comment]
Guys, I reworded the instruction per @bep's recommendation - if you would prefer it be removed entirely, let me know.  Thanks.

**Comment by scottcwilson** [comment]
Checking back to see if this is ok.

**Comment by bep** [comment]
It looks good -- it's just that I do these merges in bursts when I get some spare time.

**Comment by scottcwilson** [comment]
OK Good.  Just wanted to be sure you weren't waiting on me.  Thank you for all your work on the Hugo project!

**Comment by scottcwilson** [comment]
Squashed to one commit; updated commit message per standards.

**Comment by bep** [comment]
Merged.

**Pull request closed without merging**